modules/ROOT/pages/troubleshooting-approved-members-not-created-or-emailed.adoc

```diff
@@ -0,0 +1,18 @@
+= Approved Members Not Created or Emailed
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+
+If your Winter '20 Salesforce organization has the member approval workflow active, and approving a member and refreshing the screen removes the member from the list without creating the new user or sending the new user email, troubleshoot by either setting the approver's language to English or adding the approver's language.
+
+To set the approver's language to English, configure the approver user in the `SelfRegister` component and set the locale to English.
+
+To add the approver's language:
+
+. Open Salesforce setup.
+. Open company settings.
```

Comment:
Are "company settings" and "language settings" actual UI elements? If so, they should be bold. Same for other settings in this group of steps